lib/joi/VERSION

```diff
@@ -1 +1 @@
-13.7.0
+14.3.1
```

Comment:
This package's update was regenerated using the instructions from https://github.com/Kashoo/synctos/blob/v2.7.0/lib/joi/README.md